ietfdata/datatracker.py

```diff
@@ -785,16 +799,48 @@ def __del__(self):
         self.session.close()
 
 
+    def _cache_filepath(self, resource_uri: URI) -> Path:
+        if self.cachedir is not None:
+            return Path(self.cachedir, resource_uri.uri[1:-1] + ".json")
+        else:
+            return Path(".")
```

Comment:
This looks to be only ever called when `self.cachedir` is set. Would it be better written:
```
        assert self.cachedir is not None
        return Path(self.cachedir, resource_uri.uri[1:-1] + ".json")
```

Reply:
Fixed.